woocommerce/wc-admin
# Update completed task option when task lists are retrieved

Fixes #7375 

Updates the tracked options whenever the task lists are polled.

Note that the actual tracking methods remain unchanged to allow the old task list to continue working.

### Detailed test instructions:

1. Delete the `woocommerce_task_list_tracked_completed_tasks` option in your database.
2. Send a GET request to `wp-json/wc-admin/onboarding/tasks/`
3. Check that `woocommerce_task_list_tracked_completed_tasks` has been added with your completed tasks.
4. Complete a task and send another GET request to `wp-json/wc-admin/onboarding/tasks/`
5. Make sure that the `woocommerce_task_list_tracked_completed_tasks` option contains the newly completed task in addition to the previously completed tasks.
